src/App/App.js

```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react'
-import logo from '../tomatillo3.png';
+import logo from '../images/tomatillo3.png';
```

Comment:
I can't seem to use images that are outside src -- is that weird, or is it just me??  shouldn't I be able to use images that are stored in /public??